CHANGELOG.md

```diff
@@ -0,0 +1,159 @@
+# Changelog
+All notable changes to this project made by SYNAPTICON will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+##[Unreleased]
+### EtherCAT wrapper library
```

Comment:
No no no, please don't mention the wrapper here. You can make a CHANGELOG in the wrapper folder, but eventually the whole wrapper needs to be moved.